src/sentry/lang/javascript/processor.py

```diff
@@ -187,18 +187,32 @@ def fetch_release_file(filename, release):
     cache_key = 'releasefile:v1:%s:%s' % (
         release.id,
         md5(filename).hexdigest(),
-    )
+    ),
+
+    filename_path = None
+    try:
+        # Can we guarantee filename is always parsable?
```

Comment:
Yeah but I need the order of the list ... I decided to just check `filename_path != filename` – easy enough. See latest patch.